packages/playground/app/assets/js/webcomponentsjs/README.md

```diff
@@ -0,0 +1,229 @@
+[![Build Status](https://travis-ci.org/webcomponents/webcomponentsjs.svg?branch=master)](https://travis-ci.org/webcomponents/webcomponentsjs)
```

Comment:
no need to have thirdparty readme in our output assets not sure about the license

Reply:
Readme is removed. The license is BSD, so we need to keep the license.md